apt_select/arguments.py

```diff
@@ -20,6 +21,9 @@ def get_args():
             "Find the fastest Ubuntu apt mirrors.\n"
             "Generate new sources.list file."
         ),
+        epilog="The exit code is 0 on success, 1 on error, and %d if "\
+        "sources.list already has the fastest\n"\
```

Comment:
I'd replace "fastest" with "chosen" here given that the chosen mirror might not necessarily be the fastest (or top ranked).